BraintreeDataCollector/src/main/java/com/braintreepayments/api/DataCollector.java

```diff
@@ -106,7 +106,7 @@ public void onResult(@Nullable String kountSessionId, @Nullable Exception error)
     /**
      * Collect device information for fraud identification purposes from PayPal only.
      *
-     * @param context Android Context
+     * @param context       Android Context
```

Comment:
nit: should we update the other inline docs below this with this same formatting?

Reply:
When it renders on the docs site it shouldn't have an effect. This was android studio auto formatting at work. It might make sense to consider some automation around auto-formatting though in the future.